bin/fetch_xml_getty_records

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
+
+# fetch_xml_getty_records - download the most recent dump of xml files from the getty research portal
+# requires wget and xq (on macos: `brew install wget python-yq`)
+
+CUR=`dirname $0`
+mkdir -p $CUR/../tmp
+cd $CUR/../tmp
+
+# download index
+RS_INDEX="https://portal.getty.edu/resources/xml_data/resourcedump.xml"
+wget -N $RS_INDEX
+
+LATEST=`xq -r .urlset.url[].loc resourcedump.xml | sed -e's/-part.\.zip//'| sort | uniq | grep -v "\d\d\d\d\d\d\d" | tail -1`
+echo $LATEST
+FILES=`xq -r .urlset.url[].loc resourcedump.xml | grep $LATEST`
+for i in $FILES; do
+  echo "$i"
+  wget -N $i
+  FN=`basename $i`
+  unzip -o $FN -x resourcedump_manifest*
+done
+
+cd $CUR
```

Comment:
This isn't needed (when a script exits, you are in the original directory)

Reply:
done.